price_calculator/src/main/resources/application.properties

```
@@ -0,0 +1,9 @@
+spring.application.name=price_calculator_service
```

Meant to be equivalent to the top-level folder name
```suggestion
spring.application.name=price_calculator
```